### DISCUSSION TITLE
Wiki links to modules not working

### DISCUSSION THREAD

**Discussion by johnarumemi** [nvim-neorg/neorg#918] [Q&A] [answered]
Hi,

I am trying to access a few of the wiki links for modules like `core.completion`, `core.concealer`, `core.integrations.treesitter` etc, and they are all returning a "You do not have permission to update this wiki. " error. Is there a misconfiguration somewhere for this repo?

**Comment by champignoom** [comment, marked as answer]
Yes it has recently been [silently destroyed](https://github.com/nvim-neorg/neorg/actions/runs/5129024803/jobs/9226258944) by the latest commit.

**Reply by johnarumemi** [reply]
thanks, seems to all be working now anyway.
